signal2html/templates/thread.html

```diff
@@ -139,6 +139,65 @@
         position: relative;
       }
 
+      .msg-reactions {
```

Comment:
I don't have a backup that uses reactions, would it be a lot of trouble to show a screenshot of what it ends up looking like? If it is a lot of trouble, nvm :)

Reply:
#
![reactions-screenshot](https://user-images.githubusercontent.com/82123207/118153756-8941c080-b416-11eb-9252-5687c4ef5ce6.png)

I'm a bit hesitant about the coloring of the reaction text, what do you think? Here is another version with white text, and also a white border:

![reactions-screenshot-2](https://user-images.githubusercontent.com/82123207/118156323-84324080-b419-11eb-8111-e7a3a949039c.png)

Reply:
Ah that looks cool, thanks for the screenshot! If I understand correctly, the pop up is only there on hover, right? I think white text with border looks a bit better, maybe the font doesn't need to be bold though?

Reply:
Yes, that's right, it goes away once the cursor leaves the tooltip area.

Here it is without bold:

![reactions-screenshot-3](https://user-images.githubusercontent.com/82123207/118172842-1ee84a80-b42d-11eb-9bc5-7be89ca5d955.png)

Reply:
Yeah I think that looks great! :+1:

Reply:
Perfect!